lib/ts/recipe/thirdparty/components/features/signInAndUpCallback/index.tsx

```diff
@@ -47,8 +47,11 @@ const SignInAndUpCallback: React.FC<PropType> = (props) => {
     const handleVerifyResponse = useCallback(
         async (response: Awaited<ReturnType<typeof verifyCode>>): Promise<void> => {
             if (response.status === "NO_EMAIL_GIVEN_BY_PROVIDER") {
-                return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                    error: "no_email_present",
+                return SuperTokens.getInstanceOrThrow().redirectToAuth({
+                    history: props.history,
+                    queryParams: {
+                        error: "no_email_present",
+                    },
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.